_rules/link-skip-block-content-7b576d.md

```diff
@@ -0,0 +1,814 @@
+
+In this [document][], the link at the start of the first complementary [block of repeated content][] skips more than just this [block of repeated content][] (it also skips the second complementary [block of repeated content][]).
+
+```html
+<html lang="en">
+	<head>
+		<title>The Three Kingdoms, Chapter 1</title>
+	</head>
+	<body>
+		<a href="/test-assets/bypass-blocks-cf77f2/chapter2.html">Read Chapter 2</a>
+
+		<aside id="about-book">
+			<a href="#main">Skip additional information</a>
+			<h1>About the book</h1>
+			<p>The Romance of the Three Kingdoms is a 14th century historical novel.</p>
+		</aside>
+		<aside id="bio-translator">
+			<h1>About the translator</h1>
+			Yu Sumei is a professor of English at East China Normal University.
+		</aside>
+
+		<main id="main">
+			<h1>Three Heroes Swear Brotherhood at a Feast in the Peach Garden</h1>
+			<p>
+				Unity succeeds division and division follows unity. One is bound to be replaced by the other after a long span
+				of time.
+			</p>
+		</main>
+	</body>
+</html>
+```
+
+#### Failed Example 9
+
+In this [document][], there is a link to skip the first complementary [block of repeated content][], but not the second one.
+
+```html
+<html lang="en">
+	<head>
+		<title>The Three Kingdoms, Chapter 1</title>
+	</head>
+	<body>
+		<a href="/test-assets/bypass-blocks-cf77f2/chapter2.html">Read Chapter 2</a>
+
+		<aside id="about-book">
+			<a href="#bio-translator">Skip additional information</a>
+			<h1>About the book</h1>
+			<p>The Romance of the Three Kingdoms is a 14th century historical novel.</p>
+		</aside>
+		<aside id="bio-translator">
+			<h1>About the translator</h1>
+			Yu Sumei is a professor of English at East China Normal University.
+		</aside>
+
+		<main id="main">
+			<h1>Three Heroes Swear Brotherhood at a Feast in the Peach Garden</h1>
+			<p>
+				Unity succeeds division and division follows unity. One is bound to be replaced by the other after a long span
+				of time.
+			</p>
+		</main>
+	</body>
+</html>
+```
+
+#### Failed Example 10
+
+In this [document][], the link to skip the complementary [block of repeated content][] has a non-descriptive name.
+
+```html
+<html lang="en">
+	<head>
+		<title>The Three Kingdoms, Chapter 1</title>
+	</head>
+	<body>
+		<a href="/test-assets/bypass-blocks-cf77f2/chapter2.html">Read Chapter 2</a>
+
+		<aside id="about-book">
+			<a href="#main">Read text</a>
+			<h1>About the book</h1>
+			<p>The Romance of the Three Kingdoms is a 14th century historical novel.</p>
+		</aside>
+
+		<main id="main">
+			<h1>Three Heroes Swear Brotherhood at a Feast in the Peach Garden</h1>
+			<p>
+				Unity suchceeds division and division follows unity. One is bound to be replaced by the other after a long span
+				of time.
+			</p>
+		</main>
+	</body>
+</html>
+```
+
+#### Failed Example 11
+
+In this [document][], the link to skip the complementary [block of repeated content][] jumps before the last [perceivable content][] in it.
```

Comment:
The focus is at the start of the last paragraph inside the `aside`. It does not match the definition of "at the end".

The reason for rejecting that is that the link does not allow the possibility to fully skip the `aside`. Typically, screen reader users would still hear that last paragraph which is bad (this could be a very long paragraph, repeated on each page, and forcing users to hear it would not satisfy 2.4.1).

The condition on perceivability is in the definition of "at the end".